### PR TITLE
sast-coverity-check: added stats for Coverity scans

### PR DESCRIPTION
**Pull request by jperezdealgaba**
Solves: https://issues.redhat.com/browse/OSH-769

Adding the stats to Coverity scans in the result's SARIF file of successful scans.

# Before you complete this pull request ...

Look for any open pull requests in the repository with the title "e2e-tests update" and 
see if there are recent e2e-tests updates that will be applicable to your change.

### CODE REVIEW

**Comment by jperezdealgaba** [comment]
This MR was discussed here: https://github.com/jperezdealgaba/build-definitions/pull/4
and we were waiting for #1411 to be merged to create this MR

**Comment by jperezdealgaba** [comment]
@tnevrlka Would you mind enabling this pipeline please?

**Comment by jperezdealgaba** [comment]
@dirgim Would you mind revieweing this? It is very related with: https://github.com/konflux-ci/build-definitions/pull/1633

File: task/sast-coverity-check-oci-ta/0.1/sast-coverity-check-oci-ta.yaml [hunk 1]
```diff
@@ -241,7 +241,32 @@ spec:
...
+        if [ "$TOTAL_FILES" -ne 0 ]; then
+          COVERAGE_RATIO=$((SUCCEEDED * 100 / TOTAL_FILES))
+        else
+          COVERAGE_RATIO="0"
+        fi
+
+        csgrep --mode=sarif --set-scan-prop cov-scanned-files-coverage:"${COVERAGE_RATIO}" \
+          --set-scan-prop cov-scanned-files-success:"${SUCCEEDED}" \
+          --set-scan-prop cov-scanned-files-total:"${TOTAL_FILES}" \
+          --set-scan-prop cov-analyzed-loc:"${LINES_OF_CODE}"
```
Comment (kdudka) [inline task/sast-coverity-check-oci-ta/0.1/sast-coverity-check-oci-ta.yaml:267]: @jperezdealgaba Could you please rename `cov-analyzed-loc` to `cov-scanned-lines` as suggested at https://github.com/jperezdealgaba/build-definitions/pull/4#issuecomment-2500809568 ?

**Review by kdudka — APPROVED** [review]

File: task/sast-coverity-check-oci-ta/0.1/sast-coverity-check-oci-ta.yaml (hunk 1 above)
Comment (jperezdealgaba) [reply task/sast-coverity-check-oci-ta/0.1/sast-coverity-check-oci-ta.yaml]: Fixed!

**Review by kdudka — APPROVED** [review]
Looks good to me.  Thanks!

**Review by dirgim — APPROVED** [review]
LGTM

**Review by kasemAlem — APPROVED** [review]

**Comment by kdudka** [comment]
@kasemAlem @dirgim Could you please check what happened that `e2e-tests` failed like this?
```
365 Skipped
--- FAIL: TestE2E (5476.37s)
FAIL
```
I do not have access to the corresponding logs myself.

**Comment by kasemAlem** [comment]
> @kasemAlem @dirgim Could you please check what happened that `e2e-tests` failed like this?
> 
> ```
> 365 Skipped
> --- FAIL: TestE2E (5476.37s)
> FAIL
> ```
> 
> I do not have access to the corresponding logs myself.

Seems like the PLR logs are removed , can I rebase and rerun then we follow result ?

**Comment by kdudka** [comment]
@kasemAlem Yes, please.  Note that I rebased this pull request 4 hours ago.  It did not help.

**Comment by kasemAlem** [comment]
LGTM

**Pull request merged**